bin/zowe-setup-certificates.sh

```diff
@@ -145,6 +145,29 @@ if [[ "${VERIFY_CERTIFICATES}" == "true" ]]; then
 fi
 echo "Creating certificates and keystores... DONE"
 
+JWT_ALIAS="jwtsecret"
+APIML_PUBLIC_KEY="${KEYSTORE_PREFIX}.${JWT_ALIAS}.pem"
+P12_PUBLIC_KEY="${KEYSTORE_PREFIX}.${JWT_ALIAS}.p12"
+if [[ "${SETUP_APIML_SSO}" == "true" ]]; then
```

Comment:
Can't you know the intent of if you should do PKCS actions based on if $PKCS11_TOKEN_NAME and $PKCS11_TOKEN_NAME are not null, rather than $SETUP_API_SSO

Reply:
You could yes